### ISSUE TITLE
DT not exported from data.table

### ISSUE THREAD

**Issue by parksw3** [epinowcast/epidist#3]
Two options... either we change to 

```
data.table:::DT
```

or move away from using data.table completely and switch to using dplyr (`mutate`?). 

Am happy to make these changes. Let me know what you think.

**Comment by pearsonca** [comment]
Seems like not using DT function is a better intermediate option than those
alternatives? It's just a convenience function for piping, but I think
square bracket piping with data.table is more idiomatic anyway.

**Comment by parksw3** [comment]
That's a good point. I haven't tried square bracket piping yet actually. I will experiment with those and can make a pull request unless @seabbs has a strong preference on using DT.

**Comment by pearsonca** [comment]
To ensure clarity, when I refer to "square bracket piping", I mean for example:

```
some_dt <- fread("somefile.csv)[, 
  a_new_col := X*Y
][, # this is the square bracket "pipe", but also the direct use of the square bracket previously is "piping" as well
  b_new_col := as.character(a_new_col)
]
```

i.e., separating stages that would be idiomatically separated by a pipe in base R instead with distinct square bracket chains.

(aside: this particular syntax support in data.table is sometimes called fluent programming, referring to allowing flow / chaining of stages of operations together)

**Comment by seabbs** [comment]
> (aside 2: pretty sure @seabbs is not a fan of this approach - i agree, it can make for maintainability issues, and requires people grok the syntax - but i think there maybe some performance benefits? might be worth checking)

This is true. I would be very happy to drop use of DT though given it still hasn't made its way to a release of `data.table`. 

I think in the first instance moving to more standard but not piped `data.table` would be great (and I have been meaning to get to this :( )

**Comment by parksw3** [comment]
I will get to it later this week, if not over the weekend. I suddenly got tied up with a major paper revision...

**Comment by parksw3** [comment]
Starting this right now... got stuck again on thesis... will work on removing pipes and using more standard `data.table` approaches
